gsa/src/web/pages/extras/trashcanpage.js

```diff
@@ -15,9 +15,8 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-import React from 'react';
-
-import {connect} from 'react-redux';
+/* eslint-disable no-shadow */
```

Comment:
Please don't disable no-shadow for the whole file. Shadowing a variable may not be obvious sometimes and can lead to strange bugs.